pymine/logic/parsers/minecraft/vec2.py

```diff
@@ -0,0 +1,20 @@
+from pymine.api.errors import ParsingError
+from pymine.api.abc import AbstractParser
+from pymine.types.player import Player
+
+class Vec2(AbstractParser):
+    def __init__(self) -> None:
+        pass
+
+    def parse(self, s: str, p:[Player, None]) -> tuple:
+        a = s.split()[0]
+        b = s.split()[1]
```

Comment:
Space out your code a bit, and store the result of `.split()` so you don't have to use it twice (a small performance thing)